src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1748,7 +1747,9 @@
   "com.minecolonies.coremod.item.milky.bread.gui": "Removes potion effects",
   "com.minecolonies.coremod.item.golden.bread.gui": "Instantly heals 2 hearts",
   "com.minecolonies.coremod.item.chorus.bread.gui": "Teleports you to the surface",
-  "com.minecolonies.coremod.item.bread.available.gui": "Only available from a MineColonies Bakery",
+  "com.minecolonies.coremod.item.available.gui": "Crafted by a MineColonies %s",
+  "com.minecolonies.coremod.item.buildlevel.gui": "Crafted by a MineColonies %s level %s or higher",
+  "com.minecolonies.coremod.item.requiresresearch.gui": "Requires Research : %s",
```

Comment:
Why the spaces around the colon?

Reply:
Mostly habit.  Will remove in the next commit.